### ISSUE TITLE
Installing Flurry-iOS-SDK 12.0.1 Crash 

### ISSUE THREAD

**Issue by SalsabeelaHasan** [flurry/flutter-flurry-sdk#13]
happen when update pods:

[!] The 'Pods-Runner' target has transitive dependencies that include statically linked binaries: (/Users/salsabeela/StudioProjects/crossplatform-mobile-apps/ios/Pods/Flurry-iOS-SDK/CrashReporter.xcframework)

**Comment by hantao-flurrydev** [comment]
Hi @SalsabeelaHasan, this is possibly because of how our native sdk was packaged in the recent releases (we released our xcframework based on static library which has crash reporter xcframework based on .framework as a dependency). We have many reporters from cocoapods and other plugins about the similar issues, and we are about to release our native sdk with new build structure to fix such issues in one week or two. So please stay tuned.

**Comment by darab-singh93** [comment]
<img width="567" alt="Screenshot 2022-06-08 at 6 25 21 PM" src="https://user-images.githubusercontent.com/44564701/172621752-7f12e824-5b3c-41f1-8f6e-2b5288d0179d.png">
Hi @hantao-flurrydev 
I am facing this same issue can you please tell when you will be able to resolve this issue.

**Comment by SalsabeelaHasan** [comment]
> Hi @SalsabeelaHasan, this is possibly because of how our native sdk was packaged in the recent releases (we released our xcframework based on static library which has crash reporter xcframework based on .framework as a dependency). We have many reporters from cocoapods and other plugins about the similar issues, and we are about to release our native sdk with new build structure to fix such issues in one week or two. So please stay tuned.

thanks dear.. I'll keep an eye on the updates you make.

**Comment by longma-tech** [comment]
So long time..... when will you fix this bug??

**Comment by dev-yakuza** [comment]
When I execute `pod install`, I got the same error.

```
[!] The 'Pods-Runner' target has transitive dependencies that include statically linked binaries: (/app/ios/Pods/Flurry-iOS-SDK/CrashReporter.xcframework)
```

**Comment by poting-oath** [comment]
@SalsabeelaHasan @longma-tech @dev-yakuza This crash is caused by the native Flurry SDK (our plugin is just a wrapper). We were waiting for the new native SDK release which was just released one week ago. We are going to release our plugin soon. Thanks for your patient!

**Comment by poting-oath** [comment]
Fix released at flutter_flurry_sdk 2.2.0.

**Comment by dev-yakuza** [comment]
@poting-oath 

Thanks for your effort. 🙇 

But, I got another error like the following, when I execute the `pod install`. 😭 

```
[!] The 'Pods-Runner' target has transitive dependencies that include statically linked binaries: (/app/ios/Pods/Flurry-iOS-SDK/CrashReporter.xcframework, /app/ios/Pods/Flurry-iOS-SDK/Flurry_Config.xcframework, /app/ios/Pods/Flurry-iOS-SDK/Flurry_Messaging.xcframework, and /app/ios/Pods/Flurry-iOS-SDK/Flurry_iOS_SDK.xcframework)
```

I executed,

```
flutter pub add flutter_flurry_sdk
cd ios
pod install
```

Do I need to do some other configurations?

**Comment by poting-oath** [comment]
@dev-yakuza Did you run `pod update` before running `pod install`?
```
$ pod update
Update all pods
Updating local specs repositories
Analyzing dependencies
Downloading dependencies
Installing Flurry-iOS-SDK (12.1.1)
...
Generating Pods project
Integrating client project
Pod installation complete! There are 2 dependencies from the Podfile and 3 total pods installed.

$ pod install
Analyzing dependencies
Downloading dependencies
Generating Pods project
Integrating client project
Pod installation complete! There are 2 dependencies from the Podfile and 3 total pods installed.
```

**Comment by poting-oath** [comment]
@dev-yakuza If your project already installed previous version of flutter_flurry_sdk before, then you can upgrade it by (instead of `add`),
```
$ flutter pub upgrade flutter_flurry_sdk
```

**Comment by dev-yakuza** [comment]
@poting-oath 

Not working 😭 

I tried,

```
flutter pub upgrade flutter_flurry_sdk
cd ios
pod install
```

And

```
flutter clean
flutter pub get
cd ios
pod install
```


pubspec.yaml

```
dependencies:
  ...
  flutter_flurry_sdk: ^2.2.0
```

pubspec.lock

```
flutter_flurry_sdk:
    dependency: "direct main"
    description:
      name: flutter_flurry_sdk
      url: "https://pub.dartlang.org"
    source: hosted
    version: "2.2.0"
```

**Comment by dev-yakuza** [comment]
@poting-oath 

`pod update` also makes the same error.

I also tried,

```
rm Podfile.lock
rm -rf Pods/ pod
pod install
```

**Comment by dev-yakuza** [comment]
@poting-oath 

I just made a sample project.

- Repository: https://github.com/dev-yakuza/flurry_error

I just did the following to make a new project with the error.

```
flutter create flurry_error
cd flurry_error
flutter pub add flutter_flurry_sdk
cd ios
pod install
```

At this time, I got the following error. (**This is not related to the current issue.**)

```
[!] CocoaPods could not find compatible versions for pod "flutter_flurry_sdk":
  In Podfile:
    flutter_flurry_sdk (from `.symlinks/plugins/flutter_flurry_sdk/ios`)

Specs satisfying the `flutter_flurry_sdk (from `.symlinks/plugins/flutter_flurry_sdk/ios`)` dependency were found, but they required a higher minimum deployment target.

[!] Automatically assigning platform `iOS` with version `9.0` on target `Runner` because no platform was specified. Please specify a platform for this target in your Podfile. See `https://guides.cocoapods.org/syntax/podfile.html#platform`.
```

So, I just uncommented and updated the `Podfile` like the following.

```
# Uncomment this line to define a global platform for your project
platform :ios, '10.0'
...
```

And then, when I execute the `pod install` command again, I got the same error.

```
[!] The 'Pods-Runner' target has transitive dependencies that include statically linked binaries: (/flurry_error/ios/Pods/Flurry-iOS-SDK/CrashReporter.xcframework, /flurry_error/ios/Pods/Flurry-iOS-SDK/Flurry_Config.xcframework, /flurry_error/ios/Pods/Flurry-iOS-SDK/Flurry_Messaging.xcframework, and /flurry_error/ios/Pods/Flurry-iOS-SDK/Flurry_iOS_SDK.xcframework)
```

So, you can see the error when you clone the sample project and execute the `pod install` command.

I hope this is helpful to debug it.

**Comment by flurrydev** [comment]
@dev-yakuza I was able to get Flurry working in my iOS Flutter app by doing the following. Please let me know if this resolves your error.

1) Add `platform :ios, '10.0'` in the Podfile.
2) Remove `use_frameworks!` from the Podfile.
3) Run `flutter precache --ios`
4) Run `pod install`

The result:

Analyzing dependencies
Downloading dependencies
Installing Flurry-iOS-SDK (12.1.1)
Installing Flutter (1.0.0)
Installing flutter_flurry_sdk (0.0.1)
Generating Pods project
Integrating client project
Pod installation complete! There are 2 dependencies from the Podfile and 3 total pods installed.

**Comment by dev-yakuza** [comment]
> @dev-yakuza I was able to get Flurry working in my iOS Flutter app by doing the following. Please let me know if this resolves your error.
> 
> 1. Add `platform :ios, '10.0'` in the Podfile.
> 2. Remove `use_frameworks!` from the Podfile.
> 3. Run `flutter precache --ios`
> 4. Run `pod install`
> 
> The result:
> 
> Analyzing dependencies Downloading dependencies Installing Flurry-iOS-SDK (12.1.1) Installing Flutter (1.0.0) Installing flutter_flurry_sdk (0.0.1) Generating Pods project Integrating client project Pod installation complete! There are 2 dependencies from the Podfile and 3 total pods installed.

@flurrydev Thanks for your info. Removing `use_frameworks!` works for me.

**Comment by mustafa-707** [comment]
@flurrydev works fine after adding 
``` s.static_framework = true ``` 

to the `flutter_flurry_sdk.podspec`

**Comment by jodymac** [comment]
Where can I find the flutter_flurry_sdk.podspec

**Comment by poting-oath** [comment]
@jodymac `ios/.symlinks/plugins/flutter_flurry_sdk/ios/flutter_flurry_sdk.podspec`

**Comment by poting-oath** [comment]
@SalsabeelaHasan @longma-tech @dev-yakuza Fix released at flutter_flurry_sdk 2.2.1.
